znvis/mesh/icosahedron.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
After going over the third one, I am realizing that we could put these into a more comprehensive parent class. I think the `create_mesh` function could be generalized and we could use class propoerties e.g. for the kind of mesh, and dict-like attributes, or an automatically generated data-class (https://docs.python.org/3/library/dataclasses.html#dataclasses.make_dataclass) based on the parameters of the specific mesh. Alternatively, we make a generic method called `instantiate_mesh` which does the vertex computation, the translation, and the rotation, and we avoid re-writing 5 times or so. What do you think?